### DISCUSSION TITLE
sudo apt install flatpack

### DISCUSSION THREAD

**Discussion by ZhaomingPsych** [sn-lab/MouseGoggles#3] [Q&A] [answered]
Is it wrong with the spelling?

**Comment by misaacson01** [comment]
@ZhaomingPsych No I don't think the spelling is wrong, it's as shown here: https://flatpak.org/setup/Debian

Are you getting any errors?

**Reply by ZhaomingPsych** [reply]
Yes, I encountered an error: E: Unable to locate package update. (I'm new to Linux.)
I believe it should be "sudo apt install flatpak" instead of "sudo apt install flatpack". This code is found in the section titled "Install the Godot game engine."

**Reply by misaacson01** [reply, marked as answer]
Oh yes, you're right. I started at that line and didn't even see it! The typo has been fixed.
